test/sanity/examples/monstersInc/monstersInc.wtest

```diff
@@ -0,0 +1,137 @@
+import asustadores.*
```

Comment:
What do you think about moving these more complete but less cohesive examples to a separate test set? We could have /sanity for the "unit" tests of how the language should behave and a /examples with the other cases.
I would really like to keep the sanities as language-focused as possible, because those are the tests we run locally after each change of the language and every second counts.
So my proposal is that anything that can't be expressed as "this feature of the language should behave like this" should be keep in a folder that we can ignore, since it's probably redundant.

Reply:
Moved `examples` folder as sibling of `sanity` folder